specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,45 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}/backupStorageRedundancies": {
+      "get": {
```

Comment:
This new resource type needs to have a response body that looks like a resource (id, name, type, etc...) and support an individual GET.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource

Reply:
Removed new API in this PR.